src/Tizen.Multimedia.Remoting/MediaController/MediaControlEnums.cs

```diff
@@ -276,6 +294,88 @@ public enum MediaControlDisplayMode
         CroppedFull
     }
 
+    /// <summary>
+    /// Specifies the code which represents the result of commnunication between client and server.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public enum MediaControlResult
+    {
+        /// <summary>
+        /// The command or the event has been successfully completed.
+        /// </summary>
+        Success,
+
+        /// <summary>
+        /// The command or the event had already been completed.
+        /// </summary>
+        AlreadyDone = 200,
+
+        /// <summary>
+        /// The command or the event is aborted by some external event (e.g. aborted play command by incoming call).
+        /// </summary>
+        Aboarted = 300,
```

Comment:
Could you check if spelling is right?
Didn't you mean "Aborted"?

Reply:
I'll fix it.